Marharyta_Miliuk/Module 3 - Java/Week 2/NewYearPresent/src/by/epam/cdp/java/nypresent/utils/InputOutputStreams.java

```diff
@@ -0,0 +1,39 @@
+package by.epam.cdp.java.nypresent.utils;
+
+import java.util.List;
+import java.util.Scanner;
+
+import by.epam.cdp.java.nypresent.beans.Sweets;
+
+public class InputOutputStreams {
+	
+	public static void printMessage (String message){//creating methods for printing strings out to the console
+		System.out.println(message);
+		System.out.close();
+	}
+	
+	public static void printCollection (List<Sweets> collection){
+		System.out.println(collection);
+		System.out.close();
+	}
+	
+	public static void printObject (Sweets object){
+		System.out.println(object);
+		System.out.close();
+	}
+	
```

Comment:
I suppose you don't need 3 separate method if the methods do the same - print Object.toString()

public static void printObject (Sweets object){
+		System.out.println(object);
+		System.out.close();
+	}
It's the same as System.out.println(object.toString());

I think you can have only one method public static void printMessage (String message)
if content of other method is the same